template/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export function plus(x: number, y: number): number {
+  return x + y
+}
```

Comment:
This is probably some debug code?

Reply:
Just to see that the jest config works, I'm planning to do this as part of build to verify that the template still works. More "complicated" examples could would be welcome.